include/librealsense2/dds/topics/notification/notification-msg.h

```diff
@@ -87,6 +88,21 @@ class notification
         motion_stream_profile profiles[MAX_MOTION_PROFILES];
     };
 
+    enum class control_result : uint16_t
```

Comment:
In the enum, it's control-response -- maybe align them so they're the same name?

Reply:
The response include the result (success/failure) and will include other fields like value of options etc.

Reply:
I mean the names: line 31 it's called a `CONTROL_RESPONSE` which is different than `control_result`, therefore confusing. They should be the same.